src/ReverseProxy/Service/Config/ConfigValidator.cs

```diff
@@ -217,11 +217,21 @@ private async ValueTask ValidateAuthorizationPolicyAsync(IList<Exception> errors
 
             if (string.Equals(AuthorizationConstants.Default, authorizationPolicyName, StringComparison.OrdinalIgnoreCase))
             {
+                var policy = await _authorizationPolicyProvider.GetPolicyAsync(authorizationPolicyName);
+                if (policy != null)
+                {
+                    errors.Add(new ArgumentException($"Authorization policy '{authorizationPolicyName}' is reserved for yarp and needs to be changed"));
```

Comment:
This isn't actually a problem with the config, is it? Default and Anonymous are expected values in config. The problem is that a policy was defined in Startup using these reserved names. I'll try to clarify the comment:
```suggestion
                    errors.Add(new ArgumentException($"The application has registered an authorization policy named '{authorizationPolicyName}' that conflicts with the reserved authorization policy name used on this route. The registered policy name needs to be changed for this route to function."));
```
Is that clearer? If so let's use it for all of them.